### ISSUE TITLE
How to get a version number

### ISSUE THREAD

**Issue by kojix2** [libui-ng/libui-ng#141]
Hi!
I recently received a request to know the version of libui / libui-ng that you are using.
I replied that I will not support this in Ruby.

https://github.com/kojix2/LibUI/issues/63

However, being able to get a version number may be useful for binding authors and users.
In fact, when I create Ruby bindings for C libraries, my first goal is often to get the version.

I looked for a while, but the version number doesn't seem to be in the code. (Actually, it might be there.)
What do you think?

**Comment by cody271** [comment]
> In fact, when I create Ruby bindings for C libraries, my first goal is often to get the version.
> 
> I looked for a while, but the version number doesn't seem to be in the code. (Actually, it might be there.)
> What do you think?
>
Correct, this is definitely an expected feature of libraries which is missing from `ui.h` API. Good catch!

**Comment by szanni** [comment]
My suggested requirements on this would be:

- compile time
- string & integer

Good references IMO would be both [ffmpeg](https://ffmpeg.org/doxygen/trunk/group__lavu__ver.html) and [sqlite](https://www.sqlite.org/c3ref/c_source_id.html).

Something akin to:

- `UI_VERSION "x.y.z"`
- `UI_VERSION_MAJOR`, `UI_VERSION_MINOR`, `UI_VERSION_PATCH`
-  `UI_VERSION_NUMBER 00x00y00z` via `UI_VERSION_MAJOR * 1000 * 1000 + UI_VERSION_MINOR * 1000 + UI_VERSION_PATCH`
